agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/AgentClassLoader.java

```diff
@@ -36,11 +36,14 @@
 
   private static final String AGENT_INITIALIZER_JAR = System.getProperty("ota.initializer.jar", "");
 
+  protected final InternalJarURLHandler internalJarURLHandler;
```

Comment:
Why extract this to a field? It is still used only inside one method, no?

Reply:
updated